chesster.js

```diff
@@ -1,13 +1,22 @@
+// extlibs
 var async = require("async");
 var Botkit = require('botkit');
```

Comment:
Botkit is now required twice - once here and once by slackjs. Any impact? If so, any way to refactor this so it's only required by slackjs?

I'm thinking it should be feasible but would mean moving the controller initialization into slackjs. It would also simplify our hears functions because controller would not need to be passed in.

Instead we would have a slack object which maintains the state that controller normally would.